include/KeywordTokenizer.h

```diff
@@ -37,7 +37,7 @@ class LPPAPI KeywordTokenizer : public Tokenizer {
 public:
     virtual bool incrementToken();
     virtual void end();
-    virtual void reset();
```

Comment:
To keep backwards compatibility, could you add:

```using Tokenizer::reset;```

This will promote `reset()` to this class.